tests/test_functions.py

```diff
@@ -455,3 +456,42 @@ def test_class_in_main_is_in_right_module():
     resp = client.get("/")
 
     assert resp.status_code == 200
+
+
+def test_function_returns_none():
+    source = TEST_FUNCTIONS_DIR / "returns_none" / "main.py"
+    target = "function"
+
+    client = create_app(target, source).test_client()
+    resp = client.get("/")
+
+    assert resp.status_code == 500
+
+
+def test_legacy_function_check_env():
+    source = TEST_FUNCTIONS_DIR / "http_check_env" / "main.py"
+    target = "function"
+
+    os.environ["ENTRY_POINT"] = target
+
+    client = create_app(target, source).test_client()
+    resp = client.post("/", json={"mode": "FUNCTION_TRIGGER_TYPE"})
+    assert resp.status_code == 200
+    assert resp.data == b"http"
+
+    resp = client.post("/", json={"mode": "FUNCTION_NAME"})
+    assert resp.status_code == 200
+    assert resp.data == b"function"
```

Comment:
```suggestion
    assert resp.data == target.decode("utf-8")
```

Reply:
I assume you meant decode resp.data here, fixed.

Reply:
Oops, I meant `target.encode("utf-8")` but either works.